python/batch-predictor/docker/base.Dockerfile

```diff
@@ -35,6 +35,11 @@ ADD https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-2.2.8.ja
     $SPARK_HOME/jars
 ADD https://repo1.maven.org/maven2/com/google/cloud/spark/spark-bigquery-with-dependencies_2.12/${SPARK_BQ_CONNECTOR_VERSION}/spark-bigquery-with-dependencies_2.12-${SPARK_BQ_CONNECTOR_VERSION}.jar \
     $SPARK_HOME/jars
+
+# Add ODPS driver jar for accessing Alibaba Cloud MaxCompute.
+ADD https://github.com/aliyun/aliyun-odps-jdbc/releases/download/v3.8.2/odps-jdbc-3.8.2-jar-with-dependencies.jar $SPARK_HOME/jars
+ADD https://github.com/caraml-dev/caraml-store/releases/download/caraml-store-spark/odps/v0.0.1/custom-dialect.jar $SPARK_HOME/jars 
```

Comment:
A little random but looks funny we're importing a jar from `caraml-store` here 😅 Maybe in the future we can move that jar somewhere else.

Reply:
This is because the jar is used in caraml-store as well, and it's a scala file so i thought that was the most appropriate place to put the source code and build.

Reply:
Yea it's just a tiny thing no worries about it 😅